backend/tests/v5/unit/middleware/dataConverter/inputs/teamspaces/projects/models/commons/groups.test.js

```diff
@@ -18,7 +18,7 @@
 const _ = require('lodash');
 const { src } = require('../../../../../../../../helper/path');
 const { generateLegacyGroup, generateUUIDString, generateRandomString } = require('../../../../../../../../helper/services');
-const { OPERATORS } = require('../../../../../../../../../../src/v5/models/groups.constants');
+const { FIELD_NAME_OPERATORS, FIELD_VALUE_OPERATORS } = require('../../../../../../../../../../src/v5/models/metadata.rules.constants');
```

Comment:
same here

Reply:
NOTE: we only need the relative path without variables if it's going into a `jest.mock()`